### ISSUE TITLE
SIA-R69 Color Contrast is incorrectly checking "offscreen" text

### ISSUE THREAD

**Issue by offsetChris** [Siteimprove/alfa#1251]
Text that is moved offscreen and/or has a height of 0 is still being checked for color contrast even though there is no visible text on screen. 

Offscreen text is often used to provide accessible names or additional text content for screen reader user experience and is a very common practice but color contrast is irrelevant. 

In this example there is no visible text in the UI, but fails this color contrast check.

```
<div style="background:#fff;">
  <p style="display:block; color:#fff; height:0; text-indent:-999em;" >Offscreen text</p>
</div>
```

**Comment by Jym77** [comment]
@offsetChris I'm marking this as fixed by #1255 for the following reasons:
- same foreground/background colors: this is what #1255 fixes.
- `height: 0`: this actually does not make the element inivisible due to the default `overflow` being visible. With `overflow-y: hidden`, we should correctly consider this as invisible 🤞 
- `text-indend: -999em`: here also, a very long text would overflow the element. The fact that the text is here small enough is something Alfa cannot detect without layout system. I'm grouping the problems related to layout in #1256 instead.
